reflex/components/typography/markdown.py

```diff
@@ -38,43 +52,18 @@ class Markdown(Component):
 
     is_default = True
 
-    # Custom defined styles for the markdown elements.
-    custom_styles: Dict[str, Style] = {
```

Comment:
we probably should deprecate this first, unless you think no one is using it or it's already broken

Reply:
Yes agreed, I will add this for the following release I think I can make the old version convert cleanly to the new one.